src/main/java/com/neighborhoodAssurance/NAController.java

```diff
@@ -3,19 +3,25 @@
 import com.neighborhoodAssurance.dto.AgencyDTO;
 import com.neighborhoodAssurance.service.IAgencyService;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.converter.xml.SourceHttpMessageConverter;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.servlet.ModelAndView;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.web.bind.annotation.RequestParam;
 
 @Controller
 public class NAController {
-	
+
+	Logger logger = LoggerFactory.getLogger(NAController.class);
+
 	@Autowired
 	private IAgencyService agencyServiceStub;
 
-	@RequestMapping(value="/home", method=RequestMethod.GET) 
+	@RequestMapping(value="/", method=RequestMethod.GET) 

```

Comment:
I fixed your base routing, when I first started your project it didn't work.

Reply:
If a default endpoint does not exist... good idea to add one.